src/BannerNewsletter/index.tsx

```diff
@@ -110,39 +72,36 @@ const BannerNewsletter: React.FC<Props> = (props: Props) => {
         <div css={styles.wrapper}>
             <div css={styles.contentContainer}>
                 <div css={styles.topLeftComponent}>
-                    <div css={localStyles.heading}>{header}</div>
-                    <NewsletterFrequency frequency={frequency} />
-                    <p css={styles.paragraph}>
-                        {body} {boldText && <span css={localStyles.bold}>{boldText}</span>}
-                    </p>
-                    {secondParagraph && (
-                        <p css={[styles.paragraph, styles.secondParagraph]}>{secondParagraph}</p>
-                    )}
-                    <CTA
+                    <div css={styles.heading}>{header}</div>
+                    <NewsletterFrequencyBlock
+                        frequency={frequency}
+                        colors={defaultNewsletterFrequencyColors}
+                    />
+                    <div css={styles.paragraph}>
+                        <p>
+                            {body} {boldText && <span css={styles.highlight}>{boldText}</span>}
+                        </p>
+                        {secondParagraph && <p css={styles.secondParagraph}>{secondParagraph}</p>}
+                    </div>
+                    <NewsletterCtaButton
                         subscribeToNewsletter={subscribeToNewsletter}
                         newsletterId={newsletterId as string}
-                        ophanComponentId={ophanComponentId}
+                        ophanComponentId={ophanComponentId as string}
                         trackClick={trackClick}
+                        newsletterCta="Sign up"
+                        colors={defaultNewsletterCtaButtonColors}
```

Comment:
If these are the default colours, should we need to pass them in? What if they lived in the component itself and were used when no overrides were provided?